pkg/cmd/stack/translate.go

```diff
@@ -208,6 +204,17 @@ func addVolumeMountsToBuiltImage(ctx context.Context, s *model.Stack, options *S
 			if okteto.IsOkteto() && !registry.IsOktetoRegistry(svc.Image) {
 				svc.Image = fmt.Sprintf("okteto.dev/%s-%s:okteto-with-volume-mounts", s.Name, name)
 			}
+			if !options.ForceBuild {
+				if _, err := registry.GetImageTagWithDigest(svc.Image); err != errors.ErrNotFound {
```

Comment:
I see that if there is an error getting the image tag, but that error is not a NotFound, we are also skipping the build. Should we fail in that case or just build the image to avoid to break the execution for an experience improvement?